### PR TITLE
Update field properties

### PR DESCRIPTION
**Pull request by matboivin**
* Add examples
* More simple explanations

### CODE REVIEW

File: guides/advanced_guides/field_properties.md [hunk 1]
```diff
@@ -1,36 +1,67 @@
...
+## Searchable Fields
 
-## Searchable fields
+The **values** of the fields whose attributes are added to the [searchable-attributes list](/references/searchable_attributes.md) are **searched for matching query words**.
 
-Searchable fields are fields whose attribute is present in the [searchable-attributes](/references/searchable_attributes.md) list.
+Their content is used by MeiliSearch to assess the relevancy of a document.
 
-The content of a searchable field is used by MeiliSearch to **assess the relevancy of a document**.
+**By default, all field attributes are added to the searchable-attributes list**. If a new document is ingested and one of its fields is missing from other documents, the latter will be automatically added to the searchable-attributes list. [This behavior can be changed](/references/accept_new_fields.md).
```
Comment (bidoubiwa) [inline guides/advanced_guides/field_properties.md:11]:
```suggestion
**By default, all field attributes are added to the searchable-attributes list**. If a new document is ingested and one of its fields was never present in any other document, the latter will be automatically added to the searchable-attributes list. [This behavior can be changed](/references/accept_new_fields.md).
```

File: guides/advanced_guides/field_properties.md [hunk 2]
```diff
@@ -1,36 +1,67 @@
...
+## Displayed Fields
 
-Documents returned upon search **contain only displayed fields**.
+The fields whose attributes are added to the [displayed-attributes list](/references/displayed_attributes.md) are **displayed in each matching document**.
 
-Therefore, when a field's attribute is not in the displayed-attribute list, the field won't be added to the documents.
+Documents returned upon search contain only displayed fields.
 
-By default, all field attributes are added to the displayed-attributes list. If a new document is ingested and one of its fields is missing from other documents, the latter will be automatically added to the displayed-attributes list. [This behavior can be changed](/references/accept_new_fields.md).
+**By default, all field attributes are added to the displayed-attributes list**. If a new document is ingested and one of its fields is missing from other documents, the latter will be automatically added to the displayed-attributes list. [This behavior can be changed](/references/accept_new_fields.md).
```
Comment (bidoubiwa) [inline guides/advanced_guides/field_properties.md:41]:
```suggestion
**By default, all field attributes are added to the displayed-attributes list**. If a new document is ingested and one of its fields was never present in any other document, the latter will be automatically added to the displayed-attributes list. [This behavior can be changed](/references/accept_new_fields.md).
```

File: guides/advanced_guides/field_properties.md [hunk 3]
```diff
@@ -1,36 +1,67 @@
...
 
-This list can be restricted to a specific set of attributes that you can select in the settings. That way, you are able to specify which fields should not be returned upon search.
+Therefore, if a field attribute is not in the displayed-attribute list, the field won't be added to the documents but will remain stored in the server. This list can be restricted to a selected set of attributes in the settings.
```
Comment (bidoubiwa) [inline guides/advanced_guides/field_properties.md:43]:
```suggestion
Therefore, if a field attribute is not in the displayed-attribute list, the field won't be added to the documents returned upon search. Nevertheless, the documents fetched using the `GET /documents` route will have those fields, this setting only affects the document hits upon search.

This list can be restricted to a selected set of attributes in the settings.
```

File: guides/advanced_guides/field_properties.md [hunk 4]
```diff
@@ -1,36 +1,67 @@
...
+          "release_date",
+          "rank"
+      ]
+  }'
+```
 
 ## Data storing
 
-All fields are stored. This behavior **cannot be changed**.
+All fields are stored. **This behavior cannot be changed**.
```
Comment (bidoubiwa) [inline guides/advanced_guides/field_properties.md:65]: Once we have accepted the PR of Thomas we should link it here.

File: guides/advanced_guides/field_properties.md (hunk 2 above)
Comment (matboivin) [reply guides/advanced_guides/field_properties.md:41]: More clear, thx!

File: guides/advanced_guides/field_properties.md (hunk 4 above)
Comment (matboivin) [reply guides/advanced_guides/field_properties.md:65]: Noted!

**Pull request merged**